test/frame.spec.js

```diff
@@ -145,6 +145,12 @@ module.exports.addTests = function({testRunner, expect}) {
       await page.evaluate(element => element.remove(), div);
       await waitForFunction;
     });
+    it('should disable timeout when its set to 0', async({page}) => {
+      let error = null;
+      const res = await page.waitForFunction(() => new Promise(res => setTimeout(() => res(42), 1000)), {timeout: 0}).catch(e => error = e);
```

Comment:
nit: let's do it 100ms to have less impact on test time.

Reply:
sure, done...